test/number.js

```diff
@@ -103,6 +103,15 @@ describe('Number types', function () {
       valid: [7, 35738787838, [null, schema.nullable()]],
       invalid: [2, null, [14, schema.min(10).min(15)]],
     });
+
+    it('should differentiate b/w -0 and 0', () => {
```

Comment:
Could you write the correct word instead of `b/w`? I assume it means `between` but I first read it as `black/white` :-) 

Same goes for line 125, 150 and line 175.